.github/common_environment.yml

```diff
@@ -5,4 +5,4 @@
   SLACK_ICON:            https://raw.githubusercontent.com/DFE-Digital/get-into-teaching-api/master/.github/image.png?size=48
   SLACK_USERNAME:        GiT Workflows
   SLACK_FOOTER:          Get Into Teaching API Service
-  SENTRY__DSN:           ""
+  Sentry__Dsn:           ""
```

Comment:
really?!

also, does it definitely want the double underscore?

Reply:
yep 🤯  in my local tests, they pass if the variable is named `Sentry__Dsn` but fail if it is named `SENTRY__DSN`. And yep, a double-underscore is required.

Reply:
how obscure. seems to relate to [this](https://github.com/getsentry/sentry-dotnet/issues/141). are we confident this isn't going to cause problems in production?

Reply:
Hi @carlosmartinez so it shouldn't affect Production as that value is read from AKS keyvaults; this value is used just in the github build pipeline. It is currently unset in the build pipeline, but a [new version of the Sentry library](https://github.com/DFE-Digital/get-into-teaching-api/pull/1415) requires it to be set to a blank string ""

Reply:
To add, in local tests where I have run:

```
export SENTRY__DSN=""
dotnet test --no-restore --verbosity normal 
...
Test Run Failed.
Total tests: 1051
     Passed: 1017
     Failed: 34
 Total time: 41.7688 Seconds
```

vs

```
export Sentry__Dsn="" 
dotnet test --no-restore --verbosity normal 
...
Test Run Successful.
Total tests: 1051
     Passed: 1051
 Total time: 5.5251 Minutes
     3>Done Building Project "/Users/martyn/development/dfe/get-into-teaching-api/GetIntoTeachingApiTests/GetIntoTeachingApiTests.csproj" (VSTest target(s)).
     1>Done Building Project "/Users/martyn/development/dfe/get-into-teaching-api/GetIntoTeachingApi.sln" (VSTest target(s)).

Build succeeded.
    0 Warning(s)
    0 Error(s)
```

Reply:
huh! life, what a journey.